### PR TITLE
docs: add task to generate postman collection and update READMEs

### PR DESCRIPTION
**Pull request by ecshreve**
## What does this do?
- add `gen-postman` npm task
- reorganize and update doc related information in the READMEs

## How was it tested?
localhost

## Is there a Github issue this is resolving?
no

## Was any impacted documentation updated to reflect this change?
:) 

## Here's a fun image for your troubles
random photo - update me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,9 +74,15 @@ Integration tests need to be ran in the API docker container for them to functio
...
 
 ## Documentation
-Public facing API documentation lives here: https://www.dnd5eapi.co/docs. 
+Public facing API documentation lives here: https://www.dnd5eapi.co/docs.
+
+We use [RapiDoc](https://mrin9.github.io/RapiDoc/index.html) to create the documentation UI from a bundled single file OpenAPI definition created as part of the `npm run build` task.
+
+Additional details and information on working with the documentation can be found in the `/swagger` directory's [README](src/swagger/README.md).
+
+### Postman
```
Comment (bagelbits) [inline README.md:83]: Should this say anything or just be a blank section?

File: src/swagger/README.md [hunk 2]
```diff
@@ -71,39 +50,62 @@ The root of our OpenAPI definition lives in `swagger.yml`. This file contains ge
...
+For example, the CLI command equivalent to the config file would be:
+
+```bash
+portman -l src/swagger/dist/openapi.yml -o collection.postman.json -t
+```
 
-- Generate pieces of documentation based on source code e.g., generate OpenAPI `SchemaObject` from a TypeScript `type` definition.
-- Add code snippet examples.
-- ...anything you want!
+## TODO
```
Comment (bagelbits) [inline src/swagger/README.md:98]: Why not just keep this near the top?

**Review by bagelbits — DISMISSED** [review]
Two comments but otherwise looks good.

File: README.md (hunk 1 above)
Comment (ecshreve) [reply README.md:83]: oh woops! i must've missed that in a copy/paste while i was reorganizing things, good catch

File: src/swagger/README.md (hunk 2 above)
Comment (ecshreve) [reply src/swagger/README.md:98]: sure! either way is fine with me, i think if we move it back up to the top i'll change the heading to `Future Improvements` 😂  i wouldn't want to scare anyone that stumbles across this readme with a big list of **TODO**s

File: portman-cli.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "local": "src/swagger/dist/openapi.yml",
+  "output": "collection.postman.json",
+  "includeTests": true,
+  "syncPostman": false,
+  "runNewman": false
+}
```
Comment (bagelbits) [inline portman-cli.json:7]:
```suggestion
}

```

**Review by bagelbits — APPROVED** [review]

**Review by bagelbits — APPROVED** [review]
Whoops. I thought I approved.

**Pull request merged**